core/src/main/java/com/linecorp/armeria/server/auth/AuthService.java

```diff
@@ -75,17 +80,23 @@ public static AuthServiceBuilder builder() {
     private final Authorizer<HttpRequest> authorizer;
     private final AuthSuccessHandler defaultSuccessHandler;
     private final AuthFailureHandler defaultFailureHandler;
+    private final MeterRegistry meterRegistry;
 
     AuthService(HttpService delegate, Authorizer<HttpRequest> authorizer,
-                AuthSuccessHandler defaultSuccessHandler, AuthFailureHandler defaultFailureHandler) {
+                AuthSuccessHandler defaultSuccessHandler, AuthFailureHandler defaultFailureHandler,
+                MeterRegistry meterRegistry) {
         super(delegate);
         this.authorizer = authorizer;
         this.defaultSuccessHandler = defaultSuccessHandler;
         this.defaultFailureHandler = defaultFailureHandler;
+        this.meterRegistry = meterRegistry;
     }
 
     @Override
     public HttpResponse serve(ServiceRequestContext ctx, HttpRequest req) throws Exception {
+        final Timer timer = MoreMeters.newTimer(meterRegistry, "authorization", Tags.empty());
+        final Timer.Sample sample = Timer.start();
```

Comment:
This is OK, but can we use two `System.nanoTime()` calls instead so we don't create an extra object for each request?

Reply:
Got it!
I have modified my implementation to use `System.nanoTime()`. 

> we don't create an extra object for each request?

That's a good point!